### PR TITLE
Add changelog entries for php-src GH-8388

### PR DESCRIPTION
**Pull request by nielsdos**
I probably butchered something at least.
They have UPGRADING entries, but let's already make a PR for them.

EDIT: called it :') but the pesky whitespace was already there >:( :p

### CODE REVIEW

File: reference/dom/domnode.xml [hunk 1]
```diff
@@ -284,6 +284,12 @@
         <methodname>DOMNode::getUserData</methodname> have been removed.
        </entry>
       </row>
+      <row>
+       <entry>8.3.0</entry>
+       <entry>
+        <varname>nodeValue</varname> no longer unescapes character references.
+       </entry>
+      </row>
```
Comment (Girgias) [inline reference/dom/domnode.xml:292]: More recent changelogs go to the top of the table so it's in reverse chronological order. :)

**Review by Girgias — COMMENTED** [review]
Minor issue but thank you!

**Review by Girgias — APPROVED** [review]

**Comment by kamil-tekiela** [comment]
Converting it to draft for the time being.

**Comment by Girgias** [comment]
@nielsdos is this relevant for 8.4?

**Pull request closed without merging**